packages/@glimmerx/modifier/index.ts

```diff
@@ -1 +1,13 @@
-export { on, action } from '@glimmer/modifier';
+import { action, on as glimmerOn } from '@glimmer/modifier';
+
+declare const Brand: unique symbol;
```

Comment:
Is there any value to having a single `Opaque` type like we've done elsewhere, rather than duplicating it? Alternatively, is there value to *not* doing that (as you have here)?

Reply:
In general I think having a single `Opaque` type is useful, but here our two uses are in different packages, and I didn't want to force a dependency just for that

Reply:
Ah, good call. 👍🏼